src/SignalR/clients/csharp/Client/test/FunctionalTests/HubConnectionTests.cs

```diff
@@ -770,7 +770,6 @@ async IAsyncEnumerable<string> clientStreamData()
         }
 
         [Theory]
-        [Flaky("https://github.com/aspnet/AspNetCore-Internal/issues/2465", FlakyOn.All)]
         [MemberData(nameof(HubProtocolsAndTransportsAndHubPaths))]
```

Comment:
nit: consider adding [Repeat(int)] here to help diagnose faster if the test is flaky.

Reply:
Talked irl